core/templates/dev/head/player/conversation_skin_directive.html

```diff
@@ -101,13 +101,17 @@
                        angular-html-bind="activeCard.interactionHtml">
                   </div>
                 </div>
-                <div ng-if="!exploration.isInteractionInline(activeCard.stateName)"
-                     style="opacity: 0.8;">
-                  <md-button class="instructions-button"
-                          ng-click="showSupplementalCardIfScreenIsNarrow()">
+                <div ng-if="!exploration.isInteractionInline(activeCard.stateName)" style="opacity: 0.8;">
+                  <div ng-if="isCurrentSupplementalCardNonempty() && !isScreenNarrowAndShowingTutorCard()" style="text-transform: uppercase; padding: 6px 12px;"> 
                     <[exploration.getInteractionInstructions(activeCard.stateName)]>
                     <i class="material-icons md-18" style="position: relative; top: 3px;">&#xE5C8;</i>
-                  </md-button>
+                  </div>
+                  <div ng-if="(!isCurrentSupplementalCardNonempty() || isScreenNarrowAndShowingTutorCard())">
+                    <md-button class="instructions-button" ng-click="showSupplementalCardIfScreenIsNarrow()">
```

Comment:
Btw, just for the record -- what you did before, with "ng-click" on a new line below the class, is OK as well (in order to avoid too-long lines), so in the future please feel free to take either approach. I mentioned this to @souravbadami.